reader-gtfs/src/main/java/com/graphhopper/reader/gtfs/GtfsStorage.java

```diff
@@ -17,6 +17,11 @@
 
 public class GtfsStorage implements GraphExtension {
 
+	public GtfsStorage() {
+		Runtime.getRuntime().addShutdownHook(new Thread(this::close));
```

Comment:
Maybe on the web-side we should call graphhopper.close via shutdownhook for all cases not just `pt`? Or do you have problems with missing close call on non-web cases?